docs/experimental-features/investigation-guide-actions.asciidoc

```diff
@@ -1,14 +1,14 @@
 [[interactive-investigation-guides]]
 == Interactive investigation guides
 
-NOTE: This feature is available for {stack} versions 8.6.0 and newer.
+IMPORTANT: This feature is available for {stack} versions 8.6.0 and newer. Additionally, interactive investigation guides are not compatible between release versions, so query buttons created in 8.6.0 won't render correctly in {version} and vice versa.
 
 Detection rule investigation guides suggest steps for triaging, analyzing, and responding to potential security issues. For custom rules, you can create an interactive investigation guide that includes buttons for launching runtime queries in <<timelines-ui,Timeline>>, using alert data and hard-coded literal values. This allows you to start detailed Timeline investigations directly from an alert using relevant data. 
 
 [role="screenshot"]
 image::images/ig-alert-flyout.png[Alert details flyout with interactive investigation guide,550]
 
-Each query button displays in parentheses the number of documents found. Click the button to automatically create the query in Timeline based on settings specified in the investigation guide.
+Each query button displays the number of event documents found (in parentheses). Click the button to automatically load the query in Timeline based on settings specified in the investigation guide.
```

Comment:
```suggestion
Each query button displays the number of event documents found in parentheses *( )*. Click the button to automatically load the query in Timeline based on settings specified in the investigation guide.
```
This is a tough one since we usually indicate symbols using parentheses! I'm not sure my suggestion is better than what you had but thought I'd put it out there

Reply:
Yeah @benironside this sentence has plagued me since last release. 😭 Now I'm thinking of just chucking "in parenthesis" altogether, because the doc count is the only number displayed on the button -- the user will still notice it even if we don't tell them it's in parentheses.
```suggestion
Each query button displays the number of event documents found. Click the button to automatically load the query in Timeline based on settings specified in the investigation guide.
```